src/psyclone/tests/psyir/transformations/intrinsics/matmul2code_trans_test.py

```diff
@@ -622,9 +627,9 @@ def test_validate_matmat_with_slices_on_rhs(fortran_reader):
     assign = psyir.walk(Assignment)[0]
     with pytest.raises(TransformationError) as excinfo:
         trans.validate(assign.rhs)
-    assert ("To use matmul2code_trans on matmul, each range on the result "
-            "variable 'result' must be a full range but found "
-            "result(2:4,2:5)" in str(excinfo.value))
+    assert (f"To use {trans.name} on matmul, each range on the result "
```

Comment:
"No f-string"

Reply:
Addressed in c4d60bc3ef52a6d7b2853dd5b3c3b01ce82996dc.